### PR TITLE
Debonair quokka: updated email error page

### PR DESCRIPTION
**Pull request by whimsicallyson**
https://www.notion.so/glitch/error-handling-for-email-login-route-49dc44f25f8d4ac9b6017c1bb0fc7152

https://debonair-quokka.glitch.me/login/email?token=a4cb0037-cf39-46bf-bfc2-6417958e4c77

also tagging @clottman in here, as I want her to be aware that I've merged the 404 page into `error.jsx` and removed the "full of stars" styling

screen shot 2018-12-03 at 3 27 53 pm

### CODE REVIEW

File: src/presenters/pages/login.jsx [hunk 1]
```diff
@@ -50,7 +50,7 @@ class LoginPage extends React.Component {
       return <Redirect to={this.props.hash ? `/#${this.props.hash}` : '/'}/>;
     } else if (this.state.error) {
       const genericDescription = "Hard to say what happened, but we couldn't log you in. Try again?";
-      return <ErrorPage title={`${this.props.provider} Login Problem`} description={this.state.errorMessage || genericDescription}/>;
+      return <EmailErrorPage title={`${this.props.provider} Login Problem`} description={this.state.errorMessage || genericDescription}/>;
```
Comment (GregWeil) [inline src/presenters/pages/login.jsx]: `EmailErrorPage` needs `api` as a prop but it isn't provided here

File: src/presenters/pages/error.jsx [hunk 2]
```diff
@@ -93,4 +27,46 @@ ErrorPage.propTypes = {
...
+        <h1>{title}</h1>
+        <p>{description}</p>
+        <a className="button button-link" href="/">Back to Glitch</a>
+      </div>
+    </main>
+  </Layout>
+);
+
+EmailErrorPage.propTypes = {
+  api: PropTypes.func.isRequired,
```
Comment (GregWeil) [inline src/presenters/pages/error.jsx]: title and description should be included in the required props

File: src/presenters/pages/login.jsx (hunk 1 above)
Comment (GregWeil) [reply src/presenters/pages/login.jsx]: Also, this error will show up for all login providers, not just email. It looks good in any case, but the window title is wrong

**Comment by ttseng** [comment]
The image is looking a little small to me - Can you give it a width of 470px?

File: src/presenters/pages/login.jsx (hunk 1 above)
Comment (whimsicallyson) [reply src/presenters/pages/login.jsx]: updated!

File: src/presenters/pages/error.jsx (hunk 2 above)
Comment (whimsicallyson) [reply src/presenters/pages/error.jsx]: added

**Pull request merged**